src/components/manage/Contents/ContentsItem.jsx

```diff
@@ -158,7 +158,7 @@ export const ContentsItemComponent = ({
                 }
               >
                 {item[index.id] !== 'None' ? (
-                  moment(item[index.id]).fromNow()
+                  moment(item[index.id]).format('YYYY-MM-DD')
```

Comment:
@iFlameing we need to translate the date format into different language-specific formats.

Reply:
is there something that I can use to resolve this issue. Because I don't have any idea how I can do it?